### PR TITLE
feature/801-update-ci-license-and-audit

### PR DESCRIPTION
**Pull request by lewisdaly**
Part of: mojaloop/project#801

Adds 2 new steps to the ci process:
- `vunerability-check`: uses npm audit to check for and resolve vulnerabilities in packages
- `audit-license`: checks for disallowed or unknown licenses using the license-scanner tool

### CODE REVIEW

**Comment by lewisdaly** [comment]
It appears that the test coverage hasn't been implemented yet.

**Comment by elnyry** [comment]
Yes, it hasn't been. This has been a donated service and only a few critical issues have been addressed. Coverage hasn't made the cut yet, unfortunately. We need to look at it soon.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "@hapi/boom": "7.4.2",
     "@hapi/good": "8.2.0",
-    "axios": "0.18.0",
+    "axios": "^0.19.0",
```
Comment (rmothilal) [inline package.json]: please remove caret

**Comment by lewisdaly** [comment]
Oops. I had fixed it, but forgot to `git push`.

**Comment by lewisdaly** [comment]
@rmothilal this one is ready to go again.

**Pull request merged**